client/src/components/dashboard/add-friends/add-friends.jsx

```diff
@@ -36,7 +39,7 @@ export const AddFriendsForm = (props) => {
     })
       .then((res) => res.json())
       .then((data) => {
-        console.log("What is the value of add???? ", add);
+        console.log("What is the value of add???? ", data);
```

Comment:
Did you want to change the description too?

Reply:
This has been deleted.